api/use_cases/__init__.py

```diff
@@ -0,0 +1,17 @@
+from api.share.responses import ResponseFailure, ResponseSuccess, Response
+from api.share.requests import ValidRequestObject
+
+
+class UseCase:
+    def execute(self, request_object: ValidRequestObject) -> Response:
+        if not request_object:
+            return ResponseFailure.build_from_none_request(self.__class__.__name__)
```

Comment:
actually, `if not request_object` doesn't necessarily mean that `request_object is None`, it's just falsy. So maybe we should either rename `build_from_none_request()` or put `if request_object is None`, depending on what's more accurate.

Reply:
I got it